packages/integration-testsuite/package.json

```diff
@@ -36,7 +36,7 @@
     "@josephg/resolvable": "^1.0.1",
     "body-parser": "^1.20.0",
     "express": "^4.18.1",
-    "graphql-http": "1.7.2",
+    "graphql-http": "^1.8.0",
```

Comment:
This package specifically is pinned since our package is exported for integration authors to use. Updates to `graphql-http` will have downstream effects on our consumers so we need version our package accordingly when we update `graphql-http` (and we don't want them accidentally getting newer minors due to the `^`).

Reply:
Understood! 👍  https://github.com/apollographql/apollo-server/pull/7180/commits/27277be5a42908ed4b5f9a62724effc42fa1f185